static/data/meetings/notes/2023-07-20/index.md

```diff
@@ -0,0 +1,277 @@
+# Podman Community Cabal Meeting Notes
+
+Attendees: Aditya Rajan, Anders F Björklund, Ashley Cui, Ed Santiago Munoz, Jake Correnti, Justin Jereza, Lokesh Mandvekar, Martin Jackson, Matt Heon, Miloslav Trmac, Mohan Boddu, Nalin Dahyabhai, Paul Holzinger, Tom Sweeney, Valentin Rothberg
```

Comment:
The markdown format is different than previous notes. This shouldn't change anything, but date field below needs a line to be added.